contexts/AutocratContext.tsx

```diff
@@ -40,27 +39,39 @@ export const useAutocrat = () => {
 };
 
 export function AutocratProvider({ children }: { children: ReactNode; }) {
-  const { endpoint } = useNetworkConfiguration();
+  const { endpoint, network } = useNetworkConfiguration();
   const provider = useProvider();
   const [programVersion, setProgramVersion] = useLocalStorage<ProgramVersion>({
     key: 'program_version',
-    defaultValue: AUTOCRAT_VERSIONS[0],
+    defaultValue: AUTOCRAT_VERSIONS[1],
     serialize: (value) => String(AUTOCRAT_VERSIONS.indexOf(value)),
     deserialize: (value) => AUTOCRAT_VERSIONS[Number(value)],
   });
   const { programId, idl } = programVersion!;
-  const dao = useMemo(
+  let dao = useMemo(
     () =>
       PublicKey.findProgramAddressSync(
         [utils.bytes.utf8.encode('WWCACOTMICMIBMHAFTTWYGHMB')],
         programId,
       )[0],
     [programId],
   );
+  // TODO: For v3 onwards we need to NOT do this...
+  if (programVersion.label === 'V0.3') {
+    // We need to get public key for the DAOs and somehow sort which one..
+    // const programAccounts = await provider.connection.getProgramAccounts(programId);
+
+    // const keysWeCareAbout = programAccounts.map((account) => account.pubkey)
+
+    // need to setup DAO as the KP
+    dao = new PublicKey('8tanoHEyJEQgaasEkv1DxN6umYNWDotbaEpuzstcEufb');
```

Comment:
This is hard coded, as I need to learn how to use await or how to do it properly. UseQuery?

Basically we need to use get program accounts with the new ID and then parse through those.....